tethys_apps/cli/scaffold_templates/app_templates/default/setup.py_tmpl

```diff
@@ -1,17 +1,23 @@
 import os
 import sys
 from setuptools import setup, find_packages
-from tethys_apps.app_installation import custom_develop_command, custom_install_command
+from tethys_apps.app_installation import find_resource_files
 
 # -- Apps Definition -- #
 app_package = '{{project}}'
 release_package = 'tethysapp-' + app_package
 app_class = '{{project}}.app:{{class_name}}'
 app_package_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'tethysapp', app_package)
 
+# -- Please list all dependencies in the init.yml file -- #
 # -- Python Dependencies -- #
 dependencies = []
```

Comment:
Remove this all together and put an empty list in the install_requires argument of the setup function.